pkg/util/cryptomaterial/certchains/signers.go

```diff
@@ -58,7 +58,7 @@ type CertificateSigner struct {
 	subCAs             map[string]*CertificateSigner
 	signedCertificates map[string]*signedCertificateInfo
 
-	caBundlePaths sets.String
+	caBundlePaths sets.String //nolint:staticcheck
```

Comment:
Up above `sets.String` was replaced with `sets.Set[string]`. Do we need to be consistent or is there a difference in how this variable is used?

Reply:
ah, missed this one, will fix